Wellthier/View Controllers/ComposeViewController.m

```diff
@@ -0,0 +1,45 @@
+//
+//  ComposeViewController.m
+//  Wellthier
+//
+//  Created by Craig Lee on 7/19/22.
+//
+
+#import "ComposeViewController.h"
+#import "Parse/Parse.h"
+
+@interface ComposeViewController ()
+
+@property (weak, nonatomic) IBOutlet UITextView *textView;
+
+@end
+
+@implementation ComposeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [[self.textView layer] setBorderColor:[[UIColor lightGrayColor] CGColor]];
+    [[self.textView layer] setBorderWidth:1.0];
+    [self setParams];
```

Comment:
setParams seems very generic naming convention, would recommend to replace with contextual name